rules/windows/persistence_user_account_added_to_privileged_group_ad.toml

```diff
@@ -15,7 +15,39 @@ index = ["winlogbeat-*", "logs-windows.*", "logs-system.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "User Added to Privileged Group in Active Directory"
-note = """## Config
+note = """## Triage and analysis
+
+### Investigating User Added to Privileged Group in Active Directory
+
+Privileged accounts and groups in Active Directory are those to which powerful rights, privileges, and permissions are
+granted that allow them to perform nearly any action in Active Directory and on domain-joined systems.
+
+Attackers can add users to privileged groups to maintain their level of access if other privileged accounts they do have
+access to get uncovered by the security team. This way, they can continue their operations even after the discovery of
+an abused account.
+
+This rule monitors events related to the addition of a user to one of the privileged groups.
```

Comment:
how about:

This rule monitors events related to a user being added to a privileged group.